customisation/filtering.md

```diff
@@ -10,17 +10,31 @@ featured: true
 
 Parallel data can be filtered manually or automatically.
 
-To filter parallel data, risky translations and obvious noise are either dropped or fixed.
+To filter parallel data, translation pairs with obvious noise and risky translations are dropped.
 
-Risky translations include, for example:
+Obvious issues that are 
+* Translations to the wrong language
+* Sentence pairs with mismatched URLs, names, or numbers
+* Sentence pairs with mismatched tags
+* Sentence pairs with mismatched length
+* Translations with broken encoding or invalid characters
+
+Some sentence pairs have issues that and more difficult to filter out.
```

Comment:
Typo

"Some sentence pairs have issues that *are* more difficult to filter out."